software/holoplay.yml

```
@@ -0,0 +1,12 @@
+name: "HoloPlay"
+website_url: "https://app.holoplay.io"
+source_code_url: "https://github.com/stephane-r/holoplay-pwa"
+description: "Web app using Invidious API for listening Youtube audio source."
+licenses:
+  - MIT
+platforms:
+  - Docker
+  - Nodejs
```

Changed order to match documentation.

```suggestion
platforms:
  - Nodejs
  - Docker
```